### PR TITLE
iOS9 Updates for the editor

### PR DESCRIPTION
**Pull request by bummytime**
This PR has a number of updates for iOS 9:

* Completely removes the `UIAlertView+Blocks` pod
* Updated all UIAlertView usage for iOS9 
* Updated the demo project file
* Updated legacy editor
* All warnings addressed except ALAsset ones...isolated in demo app only.

/cc @sendhil

### CODE REVIEW

**Comment by bummytime** [comment]
@diegoreymendez would you be able to review this PR?

File: Classes/WPEditorViewController.m [hunk 1]
```diff
@@ -1034,14 +1030,22 @@ - (void)didTouchMediaOptions
...
-         ];
+        NSString *title = NSLocalizedString(@"Unable to insert image",
+                                            @"Title of dialog notifing user they cannot insert an image in the editor's HTML mode.");
+        NSString *message = NSLocalizedString(@"You cannot insert images while editing HTML directly. Please switch back to visual mode.",
+                                              @"Body of dialog notifing user they cannot insert an image in the editor's HTML mode.");
+        UIAlertController *alertController = [UIAlertController alertControllerWithTitle:title
+                                                                       message:message
+                                                                preferredStyle:UIAlertControllerStyleAlert];
+        
+        UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:^(UIAlertAction * action) {
```
Comment (SergioEstevao) [inline Classes/WPEditorViewController.m]: The string OK should be localised.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 before_install:
-  - gem install cocoapods -v '0.35.0'
-osx_image: xcode6.4
+  - gem install cocoapods -v '0.37.2'
```
Comment (SergioEstevao) [inline .travis.yml]: Should't we be using cocoa pods 0.38.2?

File: Classes/WPEditorViewController.m (hunk 1 above)
Comment (bummytime) [inline Classes/WPEditorViewController.m]: Good catch @SergioEstevao..will fix.

**Comment by bummytime** [comment]
@SergioEstevao I fixed the two issues you mentioned. Let me know if there is anything else. Thanks!

**Comment by diegoreymendez** [comment]
When I try `pod install` I am getting this warning: ```[!] Your Podfile has had smart quotes sanitised. To avoid issues in the future, you should not use TextEdit for editing it. If you are not using TextEdit, you should turn off smart quotes in your editor of choice.```

File: Example/Pods/AFNetworking/AFNetworking/AFHTTPSessionManager.h [hunk 3]
```diff
@@ -222,10 +226,10 @@
 
  @see -dataTaskWithRequest:completionHandler:
  */
-- (NSURLSessionDataTask *)PATCH:(NSString *)URLString
-                     parameters:(id)parameters
-                        success:(void (^)(NSURLSessionDataTask *task, id responseObject))success
-                        failure:(void (^)(NSURLSessionDataTask *task, NSError *error))failure;
+- (nullable NSURLSessionDataTask *)PATCH:(NSString *)URLString
+                     parameters:(nullable id)parameters
```
Comment (diegoreymendez) [inline Example/Pods/AFNetworking/AFNetworking/AFHTTPSessionManager.h]: The tab alignment seems to be off.

File: Example/Pods/AFNetworking/AFNetworking/AFHTTPSessionManager.h [hunk 4]
```diff
@@ -237,11 +241,13 @@
 
  @see -dataTaskWithRequest:completionHandler:
  */
-- (NSURLSessionDataTask *)DELETE:(NSString *)URLString
-                      parameters:(id)parameters
-                         success:(void (^)(NSURLSessionDataTask *task, id responseObject))success
-                         failure:(void (^)(NSURLSessionDataTask *task, NSError *error))failure;
+- (nullable NSURLSessionDataTask *)DELETE:(NSString *)URLString
+                      parameters:(nullable id)parameters
```
Comment (diegoreymendez) [inline Example/Pods/AFNetworking/AFNetworking/AFHTTPSessionManager.h]: The tab alignment seems to be off.

**Comment by bummytime** [comment]
By popular demand, I will be busting up this PR into smaller ones. Stay tuned.

/cc @diegoreymendez

**Pull request closed without merging**